webpack/assets/javascripts/react_app/components/Layout/__tests__/__snapshots__/LayoutHelper.test.js.snap

```diff
@@ -42,60 +37,38 @@ Array [
   Object {
     "children": Array [
       Object {
-        "name": "Environments",
+        "name": undefined,
         "type": "item",
-        "url": "/environments",
+        "url": "/nameless",
       },
       Object {
-        "name": "Architectures",
-        "type": "item",
-        "url": "/architectures",
-      },
-    ],
-    "className": "visible-xs-block",
-    "icon": "fa fa-wrench",
-    "name": "User",
-    "type": "sub_menu",
-  },
-  Object {
-    "children": Array [
-      Object {
-        "name": "Domains",
-        "type": "item",
-        "url": "/domains",
-      },
-      Object {
-        "name": "Realms",
-        "type": "item",
-        "url": "/realms",
+        "name": undefined,
+        "type": "divider",
```

Comment:
Yeah, I used the undefined menu object instead of using 2 full menu objects 
https://github.com/theforeman/foreman/blob/7f54445542b35ce4df1737a69c5e69fbd30c043a/webpack/assets/javascripts/react_app/components/Layout/Layout.fixtures.js#L162

Reply:
so it's expected... great :D